course_UvA-DL/graph-neural-networks/GNN_overview.py

```diff
@@ -0,0 +1,948 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.2
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# In this tutorial, we will discuss the application of neural networks on graphs.
+# Graph Neural Networks (GNNs) have recently gained increasing popularity in both applications and research, including domains such as social networks, knowledge graphs, recommender systems, and bioinformatics.
+# While the theory and math behind GNNs might first seem complicated, the implementation of those models is quite simple and helps in understanding the methodology.
+# Therefore, we will discuss the implementation of basic network layers of a GNN, namely graph convolutions, and attention layers.
+# Finally, we will apply a GNN on a node-level, edge-level, and graph-level tasks.
+#
+# Below, we will start by importing our standard libraries.
```

Comment:
I think this is rather description, note that description is added to the notebook

Reply:
Good point. The paragraph is now removed from the notebook and merged into the description in the `.meta.yaml` file.